src/sky.cpp

```diff
@@ -91,33 +89,33 @@ Sky::Sky(s32 id, ITextureSource *tsrc):
 			m_materials[4].Lighting = true;
 	}
 
-	for (v3f &star : m_stars) {
-		star = v3f(
-			myrand_range(-10000, 10000),
-			myrand_range(-10000, 10000),
-			myrand_range(-10000, 10000)
-		);
-		star.normalize();
-	}
+	setStarCount(200); // Prevent a segfault trying to read missing memory.
+	
+	// Configure the overlay_skybox material params here:
+	// We should never define textures here until they're needed.
 
-	m_directional_colored_fog = g_settings->getBool("directional_colored_fog");
+	for (int i = 5; i < 11; i++) {
+		m_materials[i] = mat;
+		m_materials[i].MaterialType = video::EMT_TRANSPARENT_ALPHA_CHANNEL;
+	}
+		m_directional_colored_fog = g_settings->getBool("directional_colored_fog");
```

Comment:
Misaligned indent
EDIT: Also check for newly added whitespaces in this file and remove them